BUILDING.md

```diff
@@ -29,11 +29,68 @@ This section contains package maintenance procedures in preparation for deployme
   git push --tags
   ```
 * Create a new release from the new tag on GitHub and let GitHub auto-generate release notes. Mark releases used for testing as pre-release. The release bundles the code into an archive, which we'll use for deployment.
+* Pushing a new tag will generate a new Docker image with the same tag.
```

Comment:
...once the GH action is in place

Reply:
* that starts with `v` .. or is it all tags?